### PR TITLE
Mobile solutions blurb

### PR DESCRIPTION
**Pull request by adamduren**
Needs some fine tuning with the typography but we can get to that towards the end

### CODE REVIEW

File: src/assets/img/app-lifecycle-icon.svg [hunk 1]
```diff
@@ -1,3 +1,8 @@
...
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="89" height="104" viewBox="0 0 89 104">
+  <defs>
+    <style>
+      .cls-1 { fill: #19bbf1; fill-rule: evenodd; }
```
Comment (adamduren) [inline src/assets/img/app-lifecycle-icon.svg]: We could in a future release link to a stylesheet for these SVG's

File: src/index.html [hunk 2]
```diff
@@ -55,6 +55,41 @@ <h1 class="lead">Apps to <em>empower</em> your business</h1>
 
   </div>
 
+  <div class="mobile-solutions">
+    <h2>Mobile <span class="emphasis">Solutions &amp; Applications</span></h2>
```
Comment (zacharytamas) [inline src/index.html]: Why not `em`?

Comment (adamduren) [inline src/index.html]: Good point.

Comment (adamduren) [inline src/index.html]: Actually, I remember thinking about doing it for `em` but then decided not to as we may not always want em to be `text-transformed` with the `font-weight`. I'll do it for now though.

**Pull request merged**